pkg/vcs/linux.go

```diff
@@ -383,8 +384,10 @@ func ParseMaintainersLinux(text []byte) Recipients {
 
 const configBisectTag = "# Minimized by syzkaller"
 
-func (ctx *linux) Minimize(target *targets.Target, original, baseline []byte,
-	dt debugtracer.DebugTracer, pred func(test []byte) (BisectResult, error)) ([]byte, error) {
+func (ctx *linux) Minimize(target *targets.Target, original []byte, dt debugtracer.DebugTracer,
```

Comment:
Do you want to configure `how` in syz-ci config? Why do we want to configure it?

Reply:
Actually, if we decide that we always want to try to drop sanitizers, there'd be no need for extra bisection config option. In this case, `how` would remain an internal interface between `pkg/bisect` and `Minimize`.

Reply:
Okay, that's fair.

Let `Minimize` just take the baseline config and the crash title (or a report.Report object) directly.